### PR TITLE
obs_factory: support migration

### PR DESCRIPTION
**Pull request by coolo**
Some helpers to migrate staging tools

### CODE REVIEW

**Comment by coolo** [comment]
@hennevogel took me a while

File: src/api/app/models/obs_factory/staging_project.rb [hunk 1]
```diff
@@ -53,6 +53,16 @@ def adi_staging?
       /#{ADI_NAME_PREFIX}/.match?(name)
     end
 
+    # @return [Boolean] true if the project is tracked in the new API
+    def part_of_workflow?
```
Comment (vpereira) [inline src/api/app/models/obs_factory/staging_project.rb]: `has_workflow?` or `workflow_present?` are IMO better names than `part_of_workflow?` What do you think @coolo ?

Comment (coolo) [reply src/api/app/models/obs_factory/staging_project.rb]: `has_` sounds wrong - but I'm fine with `workflow_present?` if you insist

Comment (vpereira) [reply src/api/app/models/obs_factory/staging_project.rb]: yes please. I wasn't sure about `has_` and probably rubocop will complain about it... yes `workflow_present?` is definitely better :+1:

Comment (coolo) [reply src/api/app/models/obs_factory/staging_project.rb]: if that's all..

**Comment by vpereira** [comment]
@coolo the linter is broken, but it's not related with your PR.. maybe rebase on master?

**Comment by coolo** [comment]
> @coolo the linter is broken, but it's not related with your PR.. maybe rebase on master?

This *is* master: https://circleci.com/gh/openSUSE/open-build-service

**Comment by vpereira** [comment]
> > @coolo the linter is broken, but it's not related with your PR.. maybe rebase on master?
> 
> This _is_ master: https://circleci.com/gh/openSUSE/open-build-service

please rebase it now.

**Pull request merged**